pep-0676.rst

```diff
@@ -74,38 +77,42 @@ There are several requests for additional features in reading PEPs, such as:
 * support for SVG images [3]_
 * typographic quotation marks [4]_
 * additional footer information [5]_
-* intersphinx functionality [6]_
+* Intersphinx functionality [6]_
 
 These are "easy wins" from this proposal, and would serve to improve the
 quality-of-life for consumers of PEPs (including reviewers and writers). For
-example, the reference implementation no longer requires a scheduled render
+example, the new reference implementation no longer requires a scheduled render
 process to run, instead initiating rendering on every commit to the
 `python/peps`_ repository.
 
-Equally, there are a small number of broken items, for example list styles not
-being respected or support for updating images being challenging with the
-system [7]_. These would be solved by default in the proposal.
+Equally, there are a number of broken items currently; for example, list styles
+are not respected, and support for updating images is challenging with the
+current system [7]_. These are solved by the reference implementation of
+this proposal.
 
-Commercial providers such as `Read the Docs`_ can additionally enhance this
-experience, for example by providing rendered previews of changes in pull
+Third-party providers such as `Read the Docs`_ can additionally enhance this
+experience; for example, by providing rendered previews of changes in pull
 requests.
 
+
 Specification
 =============
 
 The proposed specification for rendering the PEP files to HTML is as per the
 `reference implementation`_.
 
-That the HTML files should be made available under peps.python.org_. The
-rendered files may be hosted just as static files, and behind a content
-delivery network (CDN).
+The HTML files SHOULD be made available under the ``peps.python.org`` domain.
+The rendered output SHOULD be hosted as static files, and MAY be behind a
+content delivery network (CDN).
+
+The following redirect rules MUST be created from the `python.org`_ domain:
```

Comment:
"for the"?

```suggestion
The following redirect rules MUST be created for the `python.org`_ domain:
```

Reply:
You're right; the redirects _themselves_ are "from" but the redirect _rules_ are "for".

Reply:
I thought my review comment would append to this conversation, sorry! For context it was:

> No strong feelings, I supose I'd argue the redirect rules are being created *from* python.org to peps.python.org, although the purpose is *for* peps.python.org to work.